### ISSUE TITLE
Homebridge-HomeMatic in Synology under Docker

### ISSUE THREAD

**Issue by BerkTam** [thkl/homebridge-homematic#483]
Hello, 

I have installed Homebridge under Synology in a Docker Container which works fine.
Homebridge starts and I can add the it on the iPhone Home app. 

After having all items in the iPhone Home app it doesn't shows any state of the HomeMatic devices.  Any idea why?

Synology is 1515+
Homebridge is from ozone Synology docker image built but same issue with 
the Homebridge Synology docker image from marcoraddatz

**Comment by thkl** [comment]
Please make sure that port 9090 of the Homebridge installation is reachable from your ccu.

**Comment by BerkTam** [comment]
Yes, I know and it is open, also 9091 and 9092.

**Comment by thkl** [comment]
Ok the other way to check communication is debug mode. Try end the Homebridge service and Start the Homebridge from a console with -D (Debug mode). So you are able to see incoming and outgoing network messages.  Evertime a ccu device is updating its state a event should be visible at the debug console.

**Comment by BerkTam** [comment]
ok, did.

I do get something weird like that:

![log](https://user-images.githubusercontent.com/32999183/71722997-4be52000-2e2b-11ea-928f-97fe52207970.png)

**Comment by thkl** [comment]
Oops .. ok the error is triggered by the method, which is communicating with the ccu to get value updates of devices.

At launch the homematic plugin is creating a rpc client for every interface (BidCos-RF, HmIP, Wired ...)  to communicate with the ccu, and obviously that did not happend in your case.

Please make sure the CCU Ports 2001 (BidCos-RF) 2000 (BidCos-Wired if enabled) and 2010 (HMiP) are reachable from inside your docker container ... there should also be a warning message in the log triggered by the client creation error.

**Comment by BerkTam** [comment]
Hi, thx for your quick response.

The ports 2001, 2002 are reachable. Only the 2010 was missing on my list but fixed that.

**Comment by thkl** [comment]
and there are not other error messages ?

Log should be like this : 

Creating Local HTTP Client for CCU RPC Events
CCU RPC Init Call on port 2001 for interface BidCos-RF.
....
CCU Response for init at BidCos-RF. with http://dockerIP:9090 ...Value ("") Error : (null)

Creating Local HTTP Client for CCU RPC Events
CCU RPC Init Call on port 2010 for interface HmIP-RF.
...
CCU Response for init at HmIP-RF. with http://dockerip:9092...Value ("") Error : (null)

**Comment by BerkTam** [comment]
I will try to start another new set-up. It seems something went quite wrong here.
And come back to you.

**Comment by BerkTam** [comment]
Hello Thomas, 

I tried now a fully new setup of Homebridge with Docker on a Synology and added 
only Homebridge-config-ui and Homebridge-Homematic.

From HomeMatic I have only added a Jalousie-Switch and a Light-Switch. 

What I can say so far is that the light switch does work and the state of the switch is 
shown in Home-App.

But for Jalousie and Thermostat I can use them in the Home-App and the HomeMatic
devices are also reacting but the Home-App does not show any states.

Any idea why?

thx
